app/src/main/java/com/github/swent/echo/classes/Event.kt

```diff
@@ -0,0 +1,11 @@
+package com.github.swent.echo.classes
+
+data class Event(
+    val eventId: String,
+    val organizerId: String,
+    val title: String,
+    val description: String,
+    val location: String, // TODO: Change to Location object, once it is implemented
```

Comment:
I think the location could be an other object defined by a data class. 
Or are the data classes only supposed to exactly represent what also ends up in the database? cc @octogradiste @unglazedstamp

Reply:
I'll create an issue about this for later on as I don't want this to block the merge of this one...